HelloQuad/HelloQuad.cs

```diff
@@ -0,0 +1,148 @@
+using AtomicEngine;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        Application.Run<HelloQuad>(args);
+    }
+}
+
+public class HelloQuad : AppDelegate
+{
+    // Scene reference kept here so it won't be collected by the GC
+    Scene scene;
+    Camera camera;
+    Graphics graphics;
+    Viewport viewport;
+
+    Texture2D texture;
+    VertexBuffer vertexBuffer;
+
+    public override void Start()
+    {
+        Renderer renderer = AtomicNET.GetSubsystem<Renderer>();
```

Comment:
Shouldn't need static AtomicNET.GetSubsystem here, that's for accessing subsystems outside of Atomic Objects